ocaml/testsuite/tests/typing-layouts-float64/unboxed_floats_alpha.ml

```diff
@@ -0,0 +1,118 @@
+(* TEST
+   flags = "-extension layouts_alpha"
+*)
+
+(* This file contains various tests for float#.  It's not an expect test to make
+   sure it gets tested for native code. *)
+
+(* CR layouts v2: When unboxed literals work, change this file to use them
+   instead of [of_float] on boxed literals everywhere. *)
```

Comment:
Implementing that should maybe be a separate PR, it will require work deep in the `Printf` module etc.

Reply:
I have added a task in UBT v2.5 for this.  I agree with mark this will require deep work in `porintf`, `camlFormatString`, etc.  Maybe there's some escape hatch by boxing the float silently and early, but I expect this would be a fun opportunity to really plan with layouts and gadts together in a way we haven't yet.  (It may turn out to be quite hard)